src/chapters/databases-part2/home-logic.rst

```diff
@@ -0,0 +1,62 @@
+Home Page Logic
+===============
+
+Now that we have our HTML templates ready, it's time to add some Python to get
+the game running.
+
+The ``index()`` Function
+------------------------
+
+The home page for our application is controlled by the ``index()`` function in
+``main.py``. It serves several purposes:
+
+#. To clear existing data in preparation for a new game,
+#. To process the ``Main Menu`` form submission,
+#. To hide a new batch of mines on the board,
+#. To redirect a player to the URL where they will play the game.
+
+To complete your ``index()`` function, open your Minesweeper project and code
+along with the video:
+
+.. raw:: html
+
+   <section class="vid_box">
+      <iframe class="vid" src="https://www.youtube-nocookie.com/embed/OZ1-PDXDZwI" frameborder="1" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+   </section>
+
+Video Summary
+-------------
```

Comment:
Video 4 - 1:28 min to 4:29min you are on the game_logic.py file and guessed = null 
at 2:00 you mention all set to Null, but you mention that its starting a clean slate, I don't see an issue with False as the value here if you do the black bar thingie.(no scrolling)

9:43 you go back to game_logic.py but scroll down pretty fast

Reply:
I edited the video file to cover up `NULL` and replace it with `False`.

Reply:
Looks Great!